lib/src/error_matchers.dart

```diff
@@ -28,6 +31,8 @@ const isFormatException = TypeMatcher<FormatException>();
 const isNoSuchMethodError = TypeMatcher<NoSuchMethodError>();
 
 /// A matcher for [NullThrownError].
+@Deprecated('NullThrownError is deprecated and will be removed in Dart 3. '
+    'Use `isA<TypeError>()` instead.')
 const isNullThrownError = TypeMatcher<NullThrownError>();
```

Comment:
```suggestion
const isNullThrownError = TypeMatcher<TypeError>();
```

Reply:
oh right, analysis server stops warning you about these once the definition is deprecated so I forgot to go back and fix it.